ensembl-track-api.openapi.yaml

```diff
@@ -43,15 +43,31 @@ paths:
             application/json:
               schema:
                 type: object
-                properties:
-                  track_categories:
-                    type: array
-                    items:
-                      $ref: '#/components/schemas/TrackCategory'
+                $ref: '#/components/schemas/TrackCategories'
         '400':
           description: Invalid request.
         '404':
           description: Specified genome ID was not found.
+      post:
```

Comment:
What input it expects is not reflected here ?

Reply:
Isn't `requestBody` the input?

Reply:
It is but that is the payload. Sorry I should have been clear. 
The description says `Creates new track categories for a given genome`. 
How the endpoint would know for which genome it has to create the tracks for ? 
TrackCategories payload doesn't have anything which would identify the genome ( genome_uuid ) but that is not reflected in the endpoint ( parameter section ).

Reply:
Yes, it expexts the genome id to update. Updating the spec now.